### PR TITLE
Fix redirect to course page

### PR DESCRIPTION
**Pull request by jibola2588**
## Description 📝
redirect to the courses page from home page  

## Changes 💬

I added routing to the course page when the user clicks on courses on the navigation bar

## Checklist ✅

- [ ] Reviewed the code changes thoroughly.
- [ ] Tested the changes locally and ensured they work as expected.
- [ ] Updated relevant documentation.
- [ ] Added appropriate test cases, if applicable.
- [ ] Considered potential security or performance implications.
- [ ] Code adheres to the project's style guidelines.
- [ ] Addressed all reviewer comments and concerns, if any.

## Screenshots (if applicable)
Screenshot (101)

## Additional Notes 📒

Include any additional information or context that might be helpful for the reviewers. 
***Also if you are fixing the already made PR and the reviewer demands some change in it then, in each PR you have to mention the issue you have fixed in it.***

### CODE REVIEW

**Review by lalitmee — CHANGES_REQUESTED** [review]
remove `pacakge-lock.json` changes

File: package-lock.json
Comment (jibola2588) [inline package-lock.json:1]: this file has been removed

**Comment by PiyushKalyanpy** [comment]
Your code changes should not look like this, please create a new PR instead.
image

**Comment by jibola2588** [comment]
> Your code changes should not look like this, please create a new PR instead. image

okay, I will make a new PR, and close this PR

**Pull request closed without merging**